test/datadog/lambda/trace/context.spec.rb

```diff
@@ -187,6 +191,72 @@
       )
     end
   end
+
+  context 'get_option_tags_for_function' do
+    ctx = LambdaContext.new
+    listener = Datadog::Trace::Listener.new(
+      handler_name: 'foo',
+      function_name: 'bar',
+      patch_http: true,
+      merge_xray_traces: false
+    )
+    it 'gets tags for function with no alias or version' do
+      res = listener.send(:get_option_tags, request_context: ctx, cold_start: false)
+      expect(res).to eq(
+        tags: {
+          cold_start: false,
+          function_arn: 'arn:aws:lambda:us-east-1:172597598159:function:hello-dog-ruby-dev-hello',
+          request_id: 'dcbfed85-c904-4367-bd54-984ca201ef47',
+          resource_names: 'hello-dog-ruby-dev-helloRuby25',
+          function_version: '$LATEST'
+        }
+      )
+    end
+  end
+
+  context 'get_option_tags_for_function_with_version' do
+    ctx = LambdaContextVersion.new
```

Comment:
Do you mind help update this test case, since you are here? I think `function_version` should be a number rather than `$latest`, the testing context object is a bit weird.

Reply:
Done